pandas/core/generic.py

```diff
@@ -4880,20 +4880,39 @@ def describe(self, percentiles=None, include=None, exclude=None):
 
         # median should always be included
         if (percentiles != 0.5).all():  # median isn't included
-            lh = percentiles[percentiles < .5]
-            uh = percentiles[percentiles > .5]
-            percentiles = np.hstack([lh, 0.5, uh])
-
-        def pretty_name(x):
-            x *= 100
-            if x == int(x):
-                return '%.0f%%' % x
-            else:
-                return '%.1f%%' % x
+            percentiles = np.concatenate((percentiles, [0.5]))
```

Comment:
hmm. I would do the unique check first, then add 0.5 if its not there already (because the user is passing in things, they may pass 0.5)